examples/templates/docker-with-dotfiles/main.tf

```diff
@@ -28,24 +28,26 @@ data "coder_workspace" "me" {
 }
 
 data "coder_parameter" "docker_image" {
-  name        = "What Docker image would you like to use for your workspace?"
-  description = "The Docker image will be used to build your workspace."
-  default     = "codercom/enterprise-base:ubuntu"
-  icon        = "/icon/docker.png"
-  type        = "string"
-  mutable     = false
+  name         = "docker_image"
+  display_name = "What Docker image would you like to use for your workspace?"
```

Comment:
Shouldn't that be `Docker Image`?

Reply:
What I did is,
1. Copy `name` to `display_name`
2. Set `name` to equivalent `hcl` name 
	e.g. for
	```hcl
	data "coder_parameter" "docker_image"
                                 |
  		                         v
                                name --> display_name
	```

Reply:
I see, but the overall idea is to use `display_name` for human-readable parameters.